configure.ac

```diff
@@ -255,8 +256,19 @@ AC_DEFINE_UNQUOTED([PATH_PKEXEC], ["$PKEXEC"], [Location of pkexec binary])
 AC_PATH_PROG([SUDO], [sudo], [/usr/bin/sudo], [$PATH:/usr/local/sbin:/usr/sbin:/sbin])
 AC_DEFINE_UNQUOTED([PATH_SUDO], ["$SUDO"], [Location of sudo binary])
 
-# Config
+# sh
+AC_PATH_PROG([SH], [sh], [/usr/bin/sh], [$PATH:/usr/local/sbin:/usr/sbin:/sbin])
+AC_DEFINE_UNQUOTED([PATH_SH], ["$SH"], [Location of sh binary])
```

Comment:
The path to the bourne shell is always /bin/sh. You can hard code this. We don't need to check for it. Alternatively you can ```#include <paths.h>``` and use ```_PATH_BSHELL```